src/styles/nav.scss

```
@@ -0,0 +1,44 @@
+.nav {
+  &__list {
+    display: flex;
+  }
+
+  &__item {
+    margin-left: 20px;
+  }
+
+  &__item:first-child {
+    margin-left: 0;
+  }
+
+  &__link {
+    position: relative;
+    display: inline-block;
+    padding: 23px 0;
+
+    text-transform: uppercase;
+
+    @include fontStyleNormal(14px);
+
+    color: #000;
+  }
+
+  &__link:hover {
+    color: $colorLightBlue;
+  }
+}
+
+.is-active {
+  color: $colorLightBlue;
```

```suggestion
.nav__link--is-active {
  color: $colorLightBlue;
```
В html ты добавил еще один класс `nav__link--is-active`, но в стилях забыл его поменять. Также, этот селектор потом можно будет вложить выше, в `nav` через `&`.